docuilib/plugins/uilib.js

```diff
@@ -0,0 +1,68 @@
+const path = require('path');
+const webpack = require('webpack');
+
+module.exports = ({siteDir}, options) => {
+  const baseProjectSource = [
+    path.resolve(siteDir, 'src'),
+    path.resolve(siteDir, 'node_modules/react-native-web'),
+    path.resolve(siteDir, 'node_modules/react-native-ui-lib'),
+    // // just for not getting warnings
+    path.resolve(siteDir, 'node_modules/react-native-shimmer-placeholder'),
+    path.resolve(siteDir, 'node_modules/react-native-linear-gradient')
+    // // end just for not getting warnings
+    // path.resolve(siteDir, 'node_modules/react-native-haptic-feedback'),
+    // path.resolve(siteDir, 'node_modules/react-native-animatable'),
+    // path.resolve(siteDir, 'node_modules/react-native-reanimated'),
```

Comment:
`react-native-reanimated` is installed but not added to the resolutions but it's working 🤷‍♂️